flow/connectors/postgres/cdc.go

```diff
@@ -687,6 +687,7 @@ func processMessage[Items model.Items](
 			slog.Any("CommitLSN", msg.CommitLSN),
 			slog.Any("TransactionEndLSN", msg.TransactionEndLSN))
 		batch.UpdateLatestCheckpointID(int64(msg.CommitLSN))
+		p.otelManager.Metrics.CommitLagGauge.Record(ctx, time.Now().UTC().Sub(msg.CommitTime).Microseconds())
```

Comment:
is it guaranteed to be monotonically increasing? same for MySQL
if not it's worth documenting

Reply:
For PG yes, for MySQL I'm not as sure in case replica is receiving GTIDs out of order. Drift should be minimal so not worried here